homeassistant/components/peco/sensor.py

```diff
@@ -0,0 +1,113 @@
+"""Sensor component for PECO outage counter."""
+import asyncio
+from datetime import timedelta
+from typing import Final, cast
+
+from peco import BadJSONError, HttpError
+
+from homeassistant.components.sensor import (
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import PERCENTAGE
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.aiohttp_client import async_get_clientsession
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import (
+    CoordinatorEntity,
+    DataUpdateCoordinator,
+    UpdateFailed,
+)
+
+from .const import CONF_COUNTY, DOMAIN, LOGGER, SCAN_INTERVAL
+
+PARALLEL_UPDATES: Final = 0
+SENSOR_LIST = (
```

Comment:
Adding some icons makes all the difference in the world :)

Reply:
The icon is set here: https://github.com/home-assistant/core/blob/dev/homeassistant/components/peco/sensor.py#L96